build-config/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -190,13 +190,14 @@
       <property name="message" value="When split across multiple lines, method parameters must be followed by a new line"/>
       <property name="illegalPattern" value="true"/>
     </module>
-    <module name="Regexp">
-      <!-- Matching closing argument e.g. `String foo, String bar) {` -->
-      <!-- Note: Does not match multiple arguments on their own line e.g. `String foo, String bar,` -->
-      <property name="format" value="^[^(\r\n]+[,](?![^,]*>)[^(\n\r]+\)[ ]*\{[ ]*$"/>
-      <property name="message" value="When split across multiple lines, each parameter must be placed on its own line"/>
-      <property name="illegalPattern" value="true"/>
-    </module>
+<!--    Regex incorrectly matches method parameters with multiple type params.-->
```

Comment:
Can this not be a separate PR?

Reply:
Have just removed it entirely; too painful to get working correctly.